.github/workflows/push.yml

```diff
@@ -623,7 +623,7 @@ jobs:
         run: |
           ./scripts/build-docker-container.sh $DOCKER_TAG
           echo "COMMENT<<EOF" >> $GITHUB_ENV
-          printf "Affected services are: ${{needs.prepare.outputs.IMAGES}}\n" >> $GITHUB_ENV
+          printf "Affected services are: `${{needs.prepare.outputs.IMAGES}}`\n" >> $GITHUB_ENV
```

Comment:
Why not `echo`? Prevents breakage in case `IMAGES` ever includes `%`

Reply:
probably because of the `\n` ?